gooddata-java/src/main/java/com/gooddata/sdk/service/account/AccountService.java

```diff
@@ -130,6 +133,23 @@ public Account getAccountById(final String id) {
         }
     }
 
+    /**
+     * Get account for given email
+     * @param email used as login
+     * @param organizationName (domain) in which account is present
+     * @return account for email, empty list if not found
+     * @throws GoodDataException when different error occurs
+     */
+    public Accounts getAccountByLogin(final String email, final String organizationName) {
+        notNull(email, "email");
+        notNull(organizationName, "organizationName");
+        try {
+            return restTemplate.getForObject(Account.ACCOUNT_BY_EMAIL_URI, Accounts.class, organizationName, email);
```

Comment:
Ohh, I was thinking why it returns collection. ok will address your comments soon.

Reply:
Great, thank you

Reply:
I did not throw any error on more than one, as login should be unique.

Also could I use this PR to add constructor to Account with all values - like login?

`    public Account(final String login,
                   final String email,
                   final String password,
                   final String firstName,
                   final String lastName,
                   final List<String> ipWhitelist,
                   final List<String> authenticationModes) {
        this(login, email, password, password, firstName, lastName, ipWhitelist, authenticationModes, null);
    }`

or should I create separate issue for this.

ALso please when do you expect release of both changes?

Reply:
@liry please ^ I would like to merge asap so hopefully there is release soon, thanks

Reply:
And what's the use case of such a constructor? If you're talking about some new service method, I would create separate issuer for that.

We can release new version for you right after this PR is merged ;)

Reply:
I have created separate PR https://github.com/gooddata/gooddata-java/pull/1071/files and issue.

It would be great, if you could release it together with these two changes. What we are aiming is to create user(get by login if does not exist) for given project, assign him some roles. We call it provisioning, with those 2 PR this should be possible with SDK only...